requirements.txt

```diff
@@ -1,2 +1,2 @@
 Django>=3.2
-social-auth-core >= 4.1.0
+social-auth-core >= 4.1.0,!=4.4.0
```

Comment:
If we're documenting that the min Django version is 3.2 why not `>=4.4.1` here ?

Reply:
I wanted to avoid restricting that dependency too much, so I just excluded known broken release. But `>=4.4.1` is definitely an option as well.